src/coreComponents/physicsSolvers/wavePropagation/sem/elastic/secondOrderEqn/isotropic/ElasticWaveEquationSEM.cpp

```diff
@@ -47,17 +47,17 @@ ElasticWaveEquationSEM::ElasticWaveEquationSEM( const std::string & name,
   WaveSolverBase( name,
                   parent )
 {
-  registerWrapper( viewKeyStruct::sourceConstantsString(), &m_sourceConstantsx ).
+  registerWrapper( viewKeyStruct::sourceConstantsXString(), &m_sourceConstantsx ).
```

Comment:
This is one of the true mistakes I found. The few failing integrated tests are due to this and will require a rebaseline.